src/Translation/GoogleTranslateApi.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Translation;
+
+use Google\Cloud\Core\Exception\ServiceException;
+use Google\Cloud\Translate\V2\TranslateClient;
+use Psr\Log\LoggerInterface;
+
+class GoogleTranslateApi implements TranslationApiInterface
+{
+  private const LONG_LANGUAGE_CODE = [
+    'zh-CN',
```

Comment:
Are these the only long language codes supported?

Reply:
yep